makefile

```diff
@@ -41,6 +41,11 @@ define robot_query
 robot query -i $(1) --query $(2) $(3)
 endef
 
+define robot_query_20g
+ROBOT_JAVA_ARGS='-Xmx20G' robot query -i $(1) --query $(2) $(3)
```

Comment:
The two `indirect-included` variations of CompLOINC which also included "all" sources (LOINC & LOINC-SNOMED) required at least 20G.